bookmarks/index.html

```diff
@@ -72,6 +76,14 @@ <h2 class="card__question">
                 alt="bookmark"
             /></a>
           </li>
+          <li>
+            <a href="/form/index.html" area-label="Form page"
```

Comment:
I think /form would be sufficient, wouldn't it?
```suggestion
            <a href="/form" aria-label=" Go to form page"
```